test/integration_test/testspec/migration/daemonset-with-all-components.yaml

```diff
@@ -132,7 +132,7 @@ spec:
             - name: dbusmount
               mountPath: /var/run/dbus
         - name: csi-node-driver-registrar
-          image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.1
```

Comment:
fyi we don't need to update images here, they will be updated according to the release manifest during automation runs.

Reply:
Yes I'm aware, It's better to keep these up to date in case the release manifest cannot be reached or we switch away from that model.

PR is here-
https://github.com/portworx/px-installer/pull/1512